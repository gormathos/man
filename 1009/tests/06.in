sol_id;user_id;task_id;score;posted_time
166098;1083;2003;100.00;1700945698
166099;1096;2009;16.67;1700946123
166101;1073;2003;33.33;1700948490
166104;1023;2003;11.11;1700953307
166108;1098;2005;95.00;1700956384
166113;1084;2004;89.29;1700957818
166119;1064;2004;3.57;1700961169
166126;1039;2006;20.59;1700965741
166134;1011;2001;2.78;1700966364
166143;1020;2001;100.00;1700970437
166153;1074;2005;60.00;1700972615
166164;1022;2009;12.50;1700973852
166176;1055;2002;100.00;1700974543
166189;1030;2003;11.11;1700979242
166203;1010;2005;100.00;1700985209
166218;1062;2003;81.48;1700986386
166234;1065;2004;92.86;1700986476
166251;1055;2001;47.22;1700990969
166269;1018;2002;35.90;1700993045
166288;1068;2004;32.14;1700993352
166308;1002;2010;19.23;1700997503
166329;1099;2009;91.67;1701002395
166351;1096;2007;100.00;1701007377
166374;1016;2006;23.53;1701008859
166398;1077;2004;100.00;1701010090
166423;1048;2006;17.65;1701011653
166449;1035;2007;13.16;1701016146
166476;1028;2005;15.00;1701017948
166504;1007;2001;47.22;1701019145
166533;1047;2004;17.86;1701020949
166563;1095;2004;100.00;1701022333
166594;1058;2010;100.00;1701026112
166626;1072;2007;42.11;1701030167
166659;1042;2010;38.46;1701034055
166693;1065;2008;31.43;1701038542
166728;1009;2008;100.00;1701041933
166764;1084;2004;100.00;1701046127
166801;1052;2008;100.00;1701049954
166839;1025;2005;10.00;1701054880
166878;1024;2010;57.69;1701057876
166918;1012;2010;100.00;1701063875
166959;1008;2003;100.00;1701064238
167001;1022;2001;2.78;1701064584
167044;1091;2009;25.00;1701070348
167088;1046;2010;84.62;1701070373
167133;1070;2001;100.00;1701070838
167179;1049;2009;100.00;1701071849
167226;1100;2009;100.00;1701076879
167274;1097;2007;100.00;1701077500
167323;1028;2008;31.43;1701080772
167373;1070;2009;75.00;1701085711
167424;1055;2006;50.00;1701091676
167476;1014;2001;91.67;1701094709
167529;1055;2005;20.00;1701097754
167583;1026;2008;91.43;1701103026
167638;1043;2007;52.63;1701107882
167694;1004;2009;100.00;1701110894
167751;1072;2010;92.31;1701112780
167809;1019;2005;5.00;1701112904
167868;1063;2005;100.00;1701114559
167928;1085;2004;39.29;1701115664
167989;1055;2002;17.95;1701119539
168051;1080;2008;8.57;1701120032
168114;1013;2004;100.00;1701123679
168178;1090;2004;53.57;1701127405
168243;1043;2010;80.77;1701127419
168309;1098;2004;100.00;1701129985
168376;1054;2009;100.00;1701131222
168444;1076;2007;13.16;1701133988
168513;1068;2001;100.00;1701136223
168583;1063;2004;57.14;1701136515
168654;1028;2001;100.00;1701140996
168726;1089;2007;21.05;1701143514
168799;1039;2005;60.00;1701148764
168873;1047;2007;10.53;1701151697
168948;1071;2003;44.44;1701155039
169024;1011;2001;100.00;1701156354
169101;1094;2002;94.87;1701157381
169179;1086;2005;5.00;1701159139
169258;1021;2009;12.50;1701160962
169338;1022;2008;100.00;1701161308
169419;1037;2003;40.74;1701162817
169501;1019;2009;25.00;1701167853
169584;1050;2009;20.83;1701167936
169668;1040;2005;85.00;1701173681
169753;1010;2005;95.00;1701178681
169839;1009;2002;64.10;1701182763
169926;1029;2007;100.00;1701185660
170014;1060;2001;55.56;1701188556
170103;1010;2001;16.67;1701192375
170193;1049;2008;2.86;1701195252
170284;1076;2001;13.89;1701200715
170376;1021;2007;73.68;1701204480
170469;1082;2010;96.15;1701205077
170563;1005;2003;70.37;1701206094
170658;1016;2008;100.00;1701207754
170754;1030;2004;100.00;1701213663
170851;1015;2010;57.69;1701217126
170949;1072;2002;100.00;1701218084
171048;1011;2004;25.00;1701222498
171148;1032;2004;100.00;1701225692
171249;1023;2001;41.67;1701229423
171351;1095;2009;0.00;1701231043
171454;1048;2008;42.86;1701234087
171558;1008;2001;97.22;1701234735
171663;1057;2001;47.22;1701236039
171769;1033;2003;100.00;1701241104
171876;1034;2010;84.62;1701243862
171984;1093;2004;100.00;1701245437
172093;1012;2006;29.41;1701247234
172203;1032;2003;81.48;1701249427
172314;1098;2002;76.92;1701253625
172426;1092;2004;64.29;1701258604
172539;1001;2005;80.00;1701259984
172653;1042;2004;10.71;1701265347
172768;1080;2001;91.67;1701267377
172884;1084;2008;100.00;1701269757
173001;1013;2007;100.00;1701271565
173119;1061;2001;19.44;1701272450
173238;1062;2005;65.00;1701272626
173358;1018;2001;100.00;1701273513
173479;1028;2005;50.00;1701278298
173601;1036;2008;100.00;1701283633
173724;1085;2008;20.00;1701286374
173848;1085;2002;5.13;1701288406
173973;1069;2009;75.00;1701291429
174099;1007;2001;30.56;1701296335
174226;1082;2009;29.17;1701300266
174354;1071;2010;100.00;1701304017
174483;1075;2001;83.33;1701307618
174613;1099;2003;100.00;1701312345
174744;1099;2009;87.50;1701318176
174876;1062;2003;88.89;1701321591
175009;1013;2002;97.44;1701326960
175143;1076;2008;100.00;1701331421
175278;1003;2008;28.57;1701335684
175414;1024;2007;100.00;1701341215
175551;1095;2010;100.00;1701346830
175689;1061;2001;5.56;1701348944
175828;1089;2008;40.00;1701350222
175968;1076;2005;10.00;1701350497
176109;1030;2006;50.00;1701355550
176251;1003;2005;55.00;1701358395
176394;1093;2003;59.26;1701358396
176538;1087;2009;58.33;1701359930
176683;1033;2008;100.00;1701362025
176829;1094;2003;66.67;1701364406
176976;1070;2005;40.00;1701365875
177124;1036;2010;53.85;1701371007
177273;1075;2007;39.47;1701373363
177423;1099;2010;23.08;1701376852
177574;1082;2005;95.00;1701381426
177726;1084;2008;100.00;1701381551
177879;1065;2003;100.00;1701382347
178033;1036;2007;73.68;1701382379
178188;1078;2001;27.78;1701386719
178344;1047;2009;8.33;1701391002
178501;1006;2003;14.81;1701393959
178659;1071;2002;76.92;1701395918
178818;1067;2001;100.00;1701401359
178978;1017;2004;0.00;1701404251
179139;1058;2001;100.00;1701407743
179301;1051;2009;66.67;1701410757
179464;1054;2005;100.00;1701411150
179628;1018;2001;55.56;1701412882
179793;1078;2003;81.48;1701415860
179959;1094;2004;3.57;1701417179
180126;1005;2004;64.29;1701418096
180294;1066;2005;60.00;1701418638
180463;1040;2002;94.87;1701421409
180633;1045;2004;39.29;1701422093
180804;1015;2002;28.21;1701426710
180976;1089;2003;100.00;1701431452
181149;1048;2001;69.44;1701431644
181323;1016;2006;100.00;1701436193
181498;1016;2002;46.15;1701441020
181674;1064;2007;23.68;1701444678
181851;1095;2005;80.00;1701448453
182029;1087;2001;11.11;1701451906
182208;1026;2007;71.05;1701456670
182388;1082;2010;69.23;1701461394
182569;1076;2001;100.00;1701465339
182751;1093;2006;73.53;1701470697
182934;1094;2005;65.00;1701475550
183118;1100;2004;100.00;1701478665
183303;1034;2010;53.85;1701479542
183489;1045;2004;100.00;1701480518
183676;1053;2001;13.89;1701484577
183864;1074;2006;100.00;1701489785
184053;1059;2003;29.63;1701494838
184243;1076;2001;100.00;1701499548
184434;1051;2002;46.15;1701504671
184626;1100;2010;100.00;1701506108
184819;1036;2004;46.43;1701506746
185013;1049;2001;80.56;1701510875
185208;1024;2001;100.00;1701515013
185404;1006;2001;63.89;1701515996
185601;1003;2001;100.00;1701516780
185799;1061;2010;92.31;1701517296
185998;1018;2004;92.86;1701517571
186198;1064;2001;44.44;1701523409
186399;1024;2002;100.00;1701526368
186601;1023;2005;80.00;1701530750
186804;1012;2010;26.92;1701533962
187008;1031;2002;100.00;1701535206
187213;1028;2005;85.00;1701540980
187419;1049;2002;48.72;1701544005
187626;1046;2005;45.00;1701548197
187834;1010;2001;100.00;1701550253
188043;1045;2007;21.05;1701550619
188253;1077;2010;69.23;1701555963
188464;1078;2003;88.89;1701561489
188676;1025;2003;48.15;1701562983
188889;1052;2001;100.00;1701565659
189103;1062;2005;0.00;1701570974
189318;1069;2001;80.56;1701572934
189534;1054;2001;94.44;1701576632
189751;1069;2002;17.95;1701581234
189969;1044;2004;53.57;1701581588
190188;1088;2005;100.00;1701582625
190408;1083;2003;11.11;1701584960
190629;1039;2008;40.00;1701588743
190851;1007;2005;10.00;1701594501
191074;1007;2004;82.14;1701600213
191298;1045;2001;77.78;1701605780
191523;1030;2002;41.03;1701609937
191749;1064;2010;73.08;1701612859
191976;1046;2007;100.00;1701614075
192204;1016;2001;100.00;1701616373
192433;1039;2006;100.00;1701618076
192663;1010;2003;66.67;1701618514
192894;1030;2004;100.00;1701622943
193126;1040;2010;15.38;1701626408
193359;1075;2003;100.00;1701632303
193593;1059;2004;100.00;1701634666
193828;1020;2010;23.08;1701640356
194064;1010;2006;88.24;1701642319
194301;1068;2003;48.15;1701642861
194539;1080;2004;21.43;1701643526
194778;1039;2009;58.33;1701646215
195018;1057;2006;73.53;1701649929
195259;1050;2008;17.14;1701653516
195501;1093;2004;17.86;1701659069
195744;1030;2003;100.00;1701664887
195988;1062;2009;58.33;1701668487
196233;1079;2003;100.00;1701669574
196479;1063;2005;85.00;1701673414
196726;1019;2005;35.00;1701673660
196974;1088;2008;45.71;1701678100
197223;1088;2009;66.67;1701678491
197473;1040;2007;31.58;1701679230
197724;1018;2009;0.00;1701680187
197976;1048;2009;79.17;1701681591
198229;1054;2010;80.77;1701682367
198483;1098;2004;64.29;1701686494
198738;1042;2004;14.29;1701687559
198994;1014;2010;100.00;1701691143
199251;1098;2004;39.29;1701695431
199509;1060;2004;100.00;1701697471
199768;1050;2007;31.58;1701698596
200028;1011;2009;8.33;1701701364
200289;1038;2006;38.24;1701703808
200551;1088;2007;2.63;1701707000
200814;1094;2005;80.00;1701710211
201078;1049;2002;71.79;1701711993
201343;1018;2007;100.00;1701713574
201609;1003;2009;100.00;1701718597
201876;1072;2004;64.29;1701721175
202144;1058;2002;66.67;1701725572
202413;1039;2007;18.42;1701727908
202683;1077;2006;2.94;1701730538
202954;1016;2007;97.37;1701734093
203226;1049;2008;40.00;1701739195
203499;1007;2008;25.71;1701740259
203773;1006;2002;28.21;1701743225
204048;1072;2009;100.00;1701743635
204324;1043;2001;100.00;1701747577
204601;1050;2010;57.69;1701749811
204879;1069;2001;61.11;1701755802
205158;1056;2003;100.00;1701758221
205438;1032;2004;89.29;1701763277
205719;1071;2010;7.69;1701766812
206001;1097;2008;65.71;1701772435
206284;1007;2001;72.22;1701775413
206568;1059;2006;61.76;1701780061
206853;1033;2003;0.00;1701783720
207139;1023;2010;84.62;1701786111
207426;1096;2001;58.33;1701789635
207714;1081;2010;76.92;1701790010
208003;1054;2003;11.11;1701795398
208293;1030;2009;8.33;1701796068
208584;1098;2001;100.00;1701801738
208876;1071;2010;69.23;1701805433
209169;1016;2006;20.59;1701810997
209463;1019;2010;100.00;1701814700
209758;1085;2009;50.00;1701819789
210054;1092;2006;100.00;1701822785
210351;1011;2008;20.00;1701823471
210649;1005;2003;59.26;1701825507
210948;1091;2010;38.46;1701826200
211248;1088;2007;10.53;1701831948
211549;1081;2003;33.33;1701835991
211851;1026;2006;11.76;1701836452
212154;1096;2007;94.74;1701839605
212458;1047;2005;75.00;1701844425
212763;1066;2002;76.92;1701846272
213069;1020;2009;37.50;1701850281
213376;1098;2001;11.11;1701852478
213684;1010;2007;39.47;1701853074
213993;1092;2004;39.29;1701855334
214303;1001;2010;100.00;1701856857
214614;1077;2006;91.18;1701858267
214926;1041;2007;13.16;1701860981
215239;1100;2006;23.53;1701866178
215553;1003;2007;100.00;1701871193
215868;1060;2008;11.43;1701871726
216184;1055;2009;100.00;1701875060
216501;1003;2002;74.36;1701879962
216819;1069;2004;35.71;1701882434
217138;1062;2009;37.50;1701885521
217458;1074;2002;35.90;1701885992
217779;1090;2009;100.00;1701889527
218101;1085;2008;42.86;1701892071
218424;1026;2007;63.16;1701894297
218748;1020;2005;100.00;1701899859
219073;1058;2009;100.00;1701900085
219399;1051;2004;17.86;1701904257
219726;1095;2005;80.00;1701907282
220054;1027;2006;70.59;1701911047
220383;1016;2006;82.35;1701915125
220713;1048;2010;100.00;1701915424
221044;1033;2010;19.23;1701915859
221376;1094;2003;14.81;1701916769
221709;1022;2008;25.71;1701922758
222043;1033;2010;100.00;1701928247
222378;1031;2006;0.00;1701928922
222714;1046;2010;100.00;1701934593
223051;1021;2005;100.00;1701940131
223389;1099;2007;15.79;1701944151
223728;1055;2008;97.14;1701947749
224068;1086;2003;14.81;1701952990
224409;1057;2002;76.92;1701956190
224751;1010;2008;100.00;1701960262
225094;1005;2008;97.14;1701960722
225438;1015;2007;100.00;1701965929
225783;1057;2007;44.74;1701970922
226129;1059;2006;20.59;1701971825
226476;1027;2001;38.89;1701974216
226824;1021;2005;100.00;1701980054
227173;1096;2005;50.00;1701980817
227523;1069;2009;79.17;1701981421
227874;1038;2007;100.00;1701986588
228226;1087;2003;77.78;1701990118
228579;1028;2001;100.00;1701992431
228933;1004;2009;79.17;1701992660
229288;1064;2001;30.56;1701997038
229644;1007;2002;15.38;1702001596
230001;1039;2007;39.47;1702003156
230359;1069;2004;21.43;1702005408
230718;1025;2010;23.08;1702010420
231078;1012;2002;46.15;1702011416
231439;1029;2008;74.29;1702013723
231801;1040;2005;75.00;1702016631
232164;1034;2001;100.00;1702022186
232528;1030;2008;28.57;1702027739
232893;1095;2009;100.00;1702029531
233259;1005;2002;100.00;1702033134
233626;1029;2006;100.00;1702038890
233994;1070;2002;79.49;1702041246
234363;1015;2009;33.33;1702046490
234733;1096;2002;100.00;1702049228
235104;1091;2004;100.00;1702052014
235476;1067;2009;100.00;1702057541
235849;1026;2008;100.00;1702062283
236223;1100;2009;45.83;1702064446
236598;1012;2009;50.00;1702066309
236974;1086;2001;75.00;1702069773
237351;1041;2009;20.83;1702070194
237729;1080;2001;100.00;1702071395
238108;1046;2002;61.54;1702073643
238488;1031;2008;100.00;1702079177
238869;1060;2010;38.46;1702079448
239251;1049;2005;100.00;1702083407
239634;1050;2006;94.12;1702088016
240018;1030;2010;100.00;1702090895
240403;1046;2003;18.52;1702093695
240789;1086;2002;100.00;1702098465
241176;1086;2010;88.46;1702098852
241564;1091;2003;88.89;1702100543
241953;1043;2008;77.14;1702101677
242343;1096;2007;100.00;1702106025
242734;1037;2008;77.14;1702111767
243126;1026;2010;42.31;1702116979
243519;1016;2009;100.00;1702120119
243913;1002;2003;25.93;1702124298
244308;1047;2005;40.00;1702128925
244704;1048;2003;25.93;1702129857
245101;1019;2008;91.43;1702130307
245499;1046;2010;0.00;1702135842
245898;1098;2007;73.68;1702138527
246298;1003;2009;100.00;1702142050
246699;1025;2008;42.86;1702146298
247101;1046;2007;7.89;1702148417
247504;1086;2002;43.59;1702150152
247908;1030;2001;91.67;1702154446
248313;1054;2002;100.00;1702155543
