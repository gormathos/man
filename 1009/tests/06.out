113
25.11.2023 20:54:58
26.11.2023 03:47:17
26.11.2023 04:55:43
26.11.2023 07:53:29
26.11.2023 14:02:57
26.11.2023 14:48:10
26.11.2023 18:12:13
26.11.2023 19:15:12
26.11.2023 23:38:53
27.11.2023 00:48:47
27.11.2023 01:52:34
27.11.2023 05:44:35
27.11.2023 05:50:38
27.11.2023 07:40:38
27.11.2023 07:57:29
27.11.2023 09:21:19
27.11.2023 09:31:40
27.11.2023 18:48:14
27.11.2023 19:49:19
27.11.2023 22:21:19
28.11.2023 00:06:25
28.11.2023 00:27:02
28.11.2023 01:50:23
28.11.2023 03:09:56
28.11.2023 07:25:54
28.11.2023 08:48:28
28.11.2023 15:34:20
28.11.2023 21:42:34
28.11.2023 23:21:03
29.11.2023 00:34:44
29.11.2023 02:41:32
29.11.2023 06:58:24
29.11.2023 08:10:37
29.11.2023 14:55:57
29.11.2023 15:26:05
29.11.2023 15:58:33
29.11.2023 18:47:13
30.11.2023 00:26:57
30.11.2023 02:45:45
30.11.2023 08:03:41
30.11.2023 10:46:55
30.11.2023 12:20:30
30.11.2023 16:33:45
30.11.2023 21:59:11
30.11.2023 22:12:27
01.12.2023 03:29:19
01.12.2023 05:15:43
01.12.2023 06:12:30
01.12.2023 11:50:52
01.12.2023 13:09:53
01.12.2023 21:15:39
02.12.2023 00:57:45
02.12.2023 01:28:38
02.12.2023 04:03:05
02.12.2023 06:45:48
02.12.2023 08:35:08
02.12.2023 11:03:33
02.12.2023 11:33:00
02.12.2023 14:12:48
02.12.2023 16:40:06
02.12.2023 20:50:53
03.12.2023 01:07:39
03.12.2023 05:50:25
03.12.2023 14:34:35
03.12.2023 15:12:53
03.12.2023 15:41:16
03.12.2023 17:02:23
03.12.2023 19:38:23
03.12.2023 20:17:46
04.12.2023 04:41:27
04.12.2023 05:59:34
04.12.2023 11:59:03
04.12.2023 13:44:31
04.12.2023 18:12:54
04.12.2023 19:36:37
05.12.2023 02:33:55
05.12.2023 03:39:37
05.12.2023 06:37:01
05.12.2023 18:42:18
05.12.2023 22:18:20
06.12.2023 00:33:05
06.12.2023 10:00:57
06.12.2023 13:59:53
06.12.2023 15:04:20
06.12.2023 19:05:27
06.12.2023 21:57:39
06.12.2023 22:01:25
07.12.2023 02:17:04
07.12.2023 05:50:47
07.12.2023 07:36:33
07.12.2023 09:08:51
07.12.2023 14:44:22
07.12.2023 16:18:49
07.12.2023 20:14:14
07.12.2023 22:03:08
07.12.2023 23:40:31
08.12.2023 07:56:26
08.12.2023 09:58:51
08.12.2023 10:58:54
08.12.2023 12:34:50
08.12.2023 15:27:08
08.12.2023 16:13:34
08.12.2023 17:45:41
08.12.2023 19:04:43
08.12.2023 21:36:35
08.12.2023 23:46:17
09.12.2023 00:56:47
09.12.2023 03:01:35
09.12.2023 05:07:45
09.12.2023 07:13:45
09.12.2023 11:08:39
09.12.2023 17:14:10
09.12.2023 20:59:03
